DEV/org.openl.core.annotations/src/org/openl/binding/impl/method/NonNullLiteral.java

```diff
@@ -0,0 +1,11 @@
+package org.openl.binding.impl.method;

```

Comment:
Maybe is better to change a package to something like `org.openl.annotation.binding` ?